docs/book/advanced-guide/getting-started-on-aws.md

```diff
@@ -0,0 +1,241 @@
+# AWS Cloud Guide
+
+This step-by-step guide explains how to set up and configure all the infrastructure necessary to run a ZenML pipeline on AWS.
+
+## Prerequisites
+
+- [Docker](https://www.docker.com/) installed and running.
+- [kubectl](https://kubernetes.io/docs/tasks/tools/) installed.
+- The [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) installed and authenticated.
+- ZenML and the integrations for this tutorial stack installed:
+    ```shell
+    pip install zenml
+    zenml integration install aws s3 kubernetes
+    ```
+
+## Setting up the AWS resources
+
+Let's open up a terminal which we'll use to store some values along the way which we'll need to configure our ZenML stack later.
+
+### Artifact Store (S3 bucket)
+
+{% tabs %}
+{% tab title="AWS UI" %}
+
+- Go to the [S3 website](https://s3.console.aws.amazon.com/s3/buckets).
+- Click on `Create bucket`.
+- Select a descriptive name and a region. Let's also set these as environment variables in our terminal:
+    ```shell
+    REGION=<REGION> # for example us-west-1
+    S3_BUCKET_NAME=<BUCKET_NAME>
+    ```
+
+{% tab title="AWS CLI" %}
+
+```shell
+# Replace the <PLACEHOLDERS> with a name for your bucket and the AWS region for your resources
+# Select one of the region codes for <REGION>: https://docs.aws.amazon.com/general/latest/gr/rande.html#regional-endpoints
+REGION=<REGION>  
+S3_BUCKET_NAME=<BUCKET_NAME>
+
+aws s3api create-bucket --bucket=$S3_BUCKET_NAME \
+    --region=$REGION \
+    --create-bucket-configuration=LocationConstraint=$REGION
+```
+
+{% endtab %}
+{% endtabs %}
+
+### Metadata Store (RDS MySQL database)
+
+{% tabs %}
+{% tab title="AWS UI" %}
+
+- Go to the [RDS website](https://console.aws.amazon.com/rds).
+- Make sure the correct region is selected on the top right (this region must be the same for all following steps).
+- Click on `Create database`.
+- Select `Easy Create`, `MySQL`, `Free tier` and enter values for your database name, username and password.
+- Note down the username and password:
+    ```shell
+    RDS_MYSQL_USERNAME=<RDS_MYSQL_USERNAME>
+    RDS_MYSQL_PASSWORD=<RDS_MYSQL_PASSWORD>
+    ```
+- Wait until the deployment is finished.
+- Select your new database and note down its endpoint:
+    ```shell
+    RDS_MYSQL_ENDPOINT=<RDS_MYSQL_ENDPOINT>
+    ```
+- Click on the active VPC security group, select `Inbound rules` and click on `Edit inbound rules`
+- Add a new rule with type `MYSQL/Aurora` and source `Anywhere-IPv4`.
```

Comment:
Good idea, I'll add that

Reply:
Actually when I was writing this, it came to my mind that we can't simply limit this to a certain IP right. As all the Kubernetes nodes also need to access this, this would probably require some additional configuration to get it working?

Reply:
You're right. It shouldn't be just a single IP, instead maybe the security group ID that the cluster nodes belong to.
Here, we can just go with "Feel free to restrict this to your IP address range or the desired security group ID".